File: app/src/main/java/com/organizer/layouts/calendar/DayLayout.java
package com.organizer.layouts.calendar;

import android.content.Context;
import android.view.ViewGroup;
import android.widget.TextView;
import androidx.constraintlayout.widget.ConstraintLayout;
import com.organizer.MainActivity;
import com.organizer.layouts.BaseLayout;

public class DayLayout extends ConstraintLayout
{
    private TextView textView;
    
    public DayLayout(Context context)
    {
        super(context);
        setLayoutParams(new ViewGroup.LayoutParams(
            ViewGroup.LayoutParams.WRAP_CONTENT,
            ViewGroup.LayoutParams.WRAP_CONTENT));
        
        textView = new TextView(context);
        textView.setLayoutParams(new ViewGroup.LayoutParams(
            ViewGroup.LayoutParams.MATCH_PARENT,
            ViewGroup.LayoutParams.MATCH_PARENT));
        textView.setTextAlignment(TEXT_ALIGNMENT_CENTER);
        addView(textView);
    }
    
    public TextView getTextView()
    {
        return textView;
    }
}

File: app/src/main/kotlin/com/organizer/layouts/calendar/DayLayout.kt
package com.organizer.layouts.calendar

import android.widget.TextView
import androidx.constraintlayout.widget.ConstraintLayout
import com.organizer.MainActivity.Companion.injectNow

class DayLayout : ConstraintLayout(injectNow()) {

    val textView = TextView(context)

    init {
        layoutParams = LayoutParams(
            LayoutParams.WRAP_CONTENT,
            LayoutParams.WRAP_CONTENT
        )

        addView(textView.apply {
            layoutParams = LayoutParams(
                LayoutParams.MATCH_PARENT,
                LayoutParams.MATCH_PARENT
            )
            textAlignment = TEXT_ALIGNMENT_CENTER
        })
    }
}
